sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -223,10 +223,17 @@ export class ConfigService {
   get queueConsumersPort(): number {
     return this.getCachedConfig(
       "queueConsumersPortConfig",
-      +process.env.QUEUE_CONSUMERS_PORT || 3001,
+      +process.env.QUEUE_CONSUMERS_PORT || 3002,
     );
   }
 
+  /**
+   * Queue prefix.
+   */
+  get queuePrefix(): string {
+    return this.getCachedConfig("queuePrefixConfig", process.env.QUEUE_PREFIX);
```

Comment:
actauly, this.getCachedConfig is not working as intented. I think the plan is to remove it for now. right @andrewsignori-aot